doc/nrf/glossary.rst

```diff
@@ -998,7 +998,7 @@ Glossary
       A string that identifies a project release.
 
    VPR Core
-      Pronounced "Viper Core", a :term:`Core` that is compatible with the RISC-V instruction set, meaning the industry-standard RISC-V development tools can be used.
+      A :term:`Core` (pronounced "Viper Core") that is compatible with the RISC-V instruction set, meaning the industry-standard RISC-V development tools can be used.
```

Comment:
This changes the meaning of the sentence, no?

Reply:
Actually, that's a good point. I'll come up with a better alternative